Documentation/migration-scripts/python-scripts/script_to_remove_groups.py

```
@@ -0,0 +1,173 @@
+            })
+
+    return memberships
+
+def remove_membership(url, session_id, membership_id):
+    try:
+        requests.delete(
+            f"{url}/permissions/membership/{membership_id}",
+            headers={"X-Metabase-Session": session_id},
+            timeout=10
+        ).raise_for_status()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)